diff --git a/browser/components/search/test/browser.ini b/browser/components/search/test/browser.ini
--- a/browser/components/search/test/browser.ini
+++ b/browser/components/search/test/browser.ini
@@ -21,4 +21,6 @@ support-files =
 [browser_amazon_behavior.js]
+skip-if = true # mozilla/activity-stream#2339
 [browser_bing.js]
 [browser_bing_behavior.js]
+skip-if = true # mozilla/activity-stream#2339
 [browser_contextmenu.js]
@@ -28,2 +30,3 @@ skip-if = os == "mac" # bug 967013
 [browser_ddg_behavior.js]
+skip-if = true # mozilla/activity-stream#2339
 [browser_google.js]
@@ -35,3 +38,3 @@ skip-if = artifact # bug 1315953
 [browser_google_behavior.js]
-skip-if = artifact # bug 1315953
+skip-if = true # mozilla/activity-stream#2339
 [browser_healthreport.js]
@@ -45,2 +48,3 @@ skip-if = artifact # bug 1315953
 [browser_yahoo_behavior.js]
+skip-if = true # mozilla/activity-stream#2339
 [browser_abouthome_behavior.js]
